docs/hacking.rst

```diff
@@ -0,0 +1,212 @@
+Hacking on :mod:`zope.meta`
+===========================
+
+
+Getting the Code
+################
+
+The main repository for :mod:`zope.meta` is in the Zope Foundation
+Github repository:
+
+  https://github.com/zopefoundation/zope.meta
```

Comment:
Do you plan to rename the repository after merging this?

I'm :+1: for that idea.